tests/unittests/CoreGraphics/CGGeometryTests.mm

```diff
@@ -52,4 +52,122 @@
     CGSize inverseTransformedSize = CGSizeApplyAffineTransform(transformedSize, CGAffineTransformInvert(rotate));
     EXPECT_NEAR(size.width, inverseTransformedSize.width, 0.001);
     EXPECT_NEAR(size.height, inverseTransformedSize.height, 0.001);
-}
\ No newline at end of file
+}
+
+class CGGeometryDivRect : public ::testing::TestWithParam<::testing::tuple<CGRect, CGFloat, CGRectEdge, CGRect, CGRect>> {};
+
+TEST_P(CGGeometryDivRect, RectDivide) {
+    CGRect rect = ::testing::get<0>(GetParam());
+    CGFloat amount = ::testing::get<1>(GetParam());
+    CGRectEdge edge = ::testing::get<2>(GetParam());
+    CGRect rSlice = ::testing::get<3>(GetParam());
+    CGRect rRemainder = ::testing::get<4>(GetParam());
+
+    CGRect slice, remainder;
+    CGRectDivide(rect, &slice, &remainder, amount, edge);
+
+    EXPECT_TRUE(CGRectEqualToRect(rSlice, slice));
```

Comment:
isn't supported

---
In reply to: [105253190](https://github.com/Microsoft/WinObjC/pull/2100#discussion_r105253190) [](ancestors = 105253190)